src/bb-themes/huraga/html/layout_default.html.twig

```diff
@@ -14,7 +14,11 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
 
     {% block opengraph %}{% endblock %}
-
+    {% if 'https://' in app.request.uri %}
+        <link rel='stylesheet' href="https://fonts.googleapis.com/css?family=PT+Sans:regular,italic,bold">
```

Comment:
Why not use //fonts.googleapis.com/css?family=PT+Sans:regular,italic,bold instead?

Reply:
Well.. because I didn't know that was an option